### DISCUSSION TITLE
I can't deploy nuxt 3 app on vps

### DISCUSSION THREAD

**Discussion by Muhammad-shaalan** [nuxt/nuxt#26181] [Questions]
I am trying to deploy nuxt 3 app using pm2
I installed pm2, node.
I build app using npm run build
I create ecosystem.config.cjs with the same content in documentation.
What should i do?

**Comment by teddygunawan** [comment]
Hello! I think you should be clearer on what your problem is, is everything running already when you are using PM2 to start the server? 
Assuming that everything is running properly but you still cannot access the server, then that problem is because you didn't expose the "local" server to the public, to do this, you can use nginx. You can try googling some nginx nuxt configuration for that.
